homeassistant/components/weatherflow_cloud/config_flow.py

```diff
@@ -33,9 +33,15 @@ class WeatherFlowCloudConfigFlow(ConfigFlow, domain=DOMAIN):
     VERSION = 1
 
     async def async_step_reauth(
-        self, user_input: Mapping[str, Any]
+        self, entry_data: Mapping[str, Any]
     ) -> ConfigFlowResult:
         """Handle a flow for reauth."""
+        return await self.async_step_reauth_confirm()
```

Comment:
Small change in behavior...
Previously, the original API token was tested, now the user is prompted for a new token without attempting the old token.
The logic with `if user_input is not None:` implies that this was the desired behavior.